lua/lazyvim/plugins/extras/lang/flutter.lua

```diff
@@ -0,0 +1,30 @@
+-- requires dart/flutter to be installed
+return {
+  recommended = {
+    ft = "dart",
+    root = { "pubspec.yaml" },
+  },
+  {
+    "akinsho/flutter-tools.nvim",
+    ft = "dart",
+    dependencies = {
+      "nvim-lua/plenary.nvim",
+      "stevearc/dressing.nvim",
+    },
+    config = true,
+  },
+
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = { ensure_installed = { "dart" } },
+  },
+  {
```

Comment:
Can you add a spec for coform?

Reply:
it is too unstable to add a separate spec for conform and nvim-lint because when i try to use it, without none-ls and flutter-tools.nvim, it just lags alot when lsp is analyzing and when formatter is running

Reply:
still testing , can't conclude if my previous statement is valid or not

Reply:
Can you try with saving like this `require("conform").format({ async = true })` (set it up with a custom keymap of yours) and see if there's any improvement. I read an issue about `dart` in conform repo getting timed out and the maintainer suggested to run it async.